File: Hayaa.BaseModel/Hayaa.BaseModel/OrderBy.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.BaseModel
{
    /// <summary>
    /// 排序元数据
    /// </summary>
   public class OrderBy
    {
        /// <summary>
        /// 数据要排序属性名称
        /// </summary>
        public String OrderByName { set; get; }
        /// <summary>
        /// 排序类型
        /// </summary>
        public OrderDirection OrderByDirection { set; get; }
        /// <summary>
        /// 排序优先级别，小数字靠前，相同数字则随机
        /// </summary>
        public int SortIndex { set; get; }
    }
}

File: Java/hayaa.basemodel/src/main/java/hayaa/basemodel/model/GridPager/OrderBy.java
package hayaa.basemodel.model.GridPager;

public class OrderBy {
    private String OrderByName;
    private OrderDirection OrderByDirection;
    private int SortIndex;

    public String getOrderByName() {
        return OrderByName;
    }

    public void setOrderByName(String orderByName) {
        OrderByName = orderByName;
    }

    public OrderDirection getOrderByDirection() {
        return OrderByDirection;
    }

    public void setOrderByDirection(OrderDirection orderByDirection) {
        OrderByDirection = orderByDirection;
    }

    public int getSortIndex() {
        return SortIndex;
    }

    public void setSortIndex(int sortIndex) {
        SortIndex = sortIndex;
    }


}
